starter-core/src/main/java/io/micronaut/starter/feature/agora/AgoraFeature.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.agora;
+
+import io.micronaut.starter.feature.Feature;
+
+public interface AgoraFeature extends Feature {
```

Comment:
just to keep up with branding, can we go with full `AgorapulseFeature` and `agorapulse` package? Thanks you

Reply:
done